CMakeLists.txt

```diff
@@ -11,7 +11,9 @@ cmake_policy(SET CMP0057 NEW)
 
 project(Umpire
   LANGUAGES CXX C
-  VERSION 0.3.5)
+  VERSION 1.0.0)
+
+set(UMPIRE_VERSION_RC "rc1")
```

Comment:
In `develop` I will make another PR to set this to be the git hash.

Reply:
@davidbeckingsale, when will this other PR be created?  After this is merged into develop?

Reply:
UM-426